paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -989,6 +989,11 @@ class ActivationMKLDNNHandler
       // else
       //   out = scale*(X + bias) = scale*X + scale*bias
       if (!bias_after_scale) beta *= alpha;
```

Comment:
Could you please make it two lines, to make it more readable

Reply:
I have tried to do that, but lint automatically changes it to an one-liner